### PR TITLE
Pins definition examples discussed in the workgroup.

### PR DESCRIPTION
**Pull request by asadate**
5 files with the different pins definition options discussed in the workgroup.

### CODE REVIEW

File: Pins-Examples/IT81202B_Pinout_Example_Option1.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "partType": "EC",
+    "manufacturer": "ITE",
+    "mpn": "IT81202E",
+    "datasheetVersion": "C0.3.2",
+    "releaseDate": null,
+    "pins": [
+        {
+            "_comment_1_ForWG":" Option1 : Use Arrays",
+            "terminal_identifier": "5",
```
Comment (rsnancollas) [inline Pins-Examples/IT81202B_Pinout_Example_Option1.json:10]: "functionProperities": [
{
"perFunctionName": "ALERT#",
"function": "SPI",
"direction": "inout", 
"polarity": "low"
},
{
"perFunctionName": "SERIRQ",
"function": "IRQ",
"direction": "in", 
"polarity": "high"
},
{
"perFunctionName": "GPM6",
"function": "GPIO",
"direction": "in", 
"polarity": "high"
}
]

File: Pins-Examples/IT81202B_Pinout_Example_Option1.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            "name": ["ALERT#","SERIRQ","GPM6"],
```
Comment (rsnancollas) [inline Pins-Examples/IT81202B_Pinout_Example_Option1.json]: "name": "ALERT#/SERIRQ/GPM6"

File: Pins-Examples/IT81202B_Pinout_Example_Option1.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            "standardized_name": null,
```
Comment (rsnancollas) [inline Pins-Examples/IT81202B_Pinout_Example_Option1.json:12]: "numberOfFunctions": 3

File: Pins-Examples/IT81202B_Pinout_Example_Option1.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            "description": "multiplexed pin",
+            "function":["SPI", "IRQ", "GPIO"],
+            "_comment_2_ForWG": "using GPIO here but the ITE datasheet has specific notations for the GPIO: PIO & EIO",
+            "direction":["inout","in","in"],
+            "polarity": ["low","high","high"],
+            "floatUnused": ["no", "no", "no"],
```
Comment (rsnancollas) [inline Pins-Examples/IT81202B_Pinout_Example_Option1.json]: "floatUnused": "no"

(not an array I think)

File: Pins-Examples/IT81202B_Pinout_Example_Option1.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    "releaseDate": null,
+    "pins": [
+        {
+            "_comment_1_ForWG":" Option1 : Use Arrays",
+            "terminal_identifier": "5",
+            "name": ["ALERT#","SERIRQ","GPM6"],
+            "standardized_name": null,
+            "description": "multiplexed pin",
+            "function":["SPI", "IRQ", "GPIO"],
+            "_comment_2_ForWG": "using GPIO here but the ITE datasheet has specific notations for the GPIO: PIO & EIO",
```
Comment (rsnancollas) [inline Pins-Examples/IT81202B_Pinout_Example_Option1.json:14]: Good point, maybe stick with GPIO for now just to keep things simple to start

File: Pins-Examples/IT81202B_Pinout_Example_Option1.json [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+            "voltageOptions": {
+                "value":[1.8, 3.3],
+                "unitText": "V",
+                "unitFactor": 1
+            },
+
+            "vihMin":{
+                "_Comment_1_ForWG": "Each voltageOption has a different value. Need to see how to best implement.",
+                "value":[1.35,2],
+                "_Comment_2_ForWG": "Note that for 1.8V, vihMin is listed as 0.75*VSUP",
```
Comment (rsnancollas) [inline Pins-Examples/IT81202B_Pinout_Example_Option1.json:47]: Good thing to discuss - generally a percentage of VDD - should we add this to the unit/value definition?

**Review by rsnancollas — COMMENTED** [review]
Added some comments based on our discussion

**Comment by asadate** [comment]
Obsolete

**Pull request closed without merging**